pkg/services/grafana-apiserver/rest/dualwriter.go

```diff
@@ -82,8 +83,15 @@ func (d *DualWriter) Create(ctx context.Context, obj runtime.Object, createValid
 		if err != nil {
 			return nil, err
 		}
-		obj = created // write the updated version
-		rsp, err := d.Storage.Create(ctx, obj, createValidation, options)
+
+		accessor, err := meta.Accessor(created)
+		if err != nil {
+			return created, err
+		}
+		accessor.SetResourceVersion("")
+		accessor.SetUID("")
```

Comment:
^^ the above is the fix to make etcd work again (and added the same error to file storage if rv is set)